src/sentry/static/sentry/app/utils/performance/histogram/types.ts

```diff
@@ -4,3 +4,5 @@ type HistogramDataUnit = {
 };
 
 export type HistogramData = HistogramDataUnit[];
```

Comment:
Slightly different from the original type to save a little typing and matches the naming of the type of an ECharts series a little better.